doc/source/cookbook.rst

```diff
@@ -1318,3 +1318,112 @@ of the data values:
        'weight': [100, 140, 180],
        'sex': ['Male', 'Female']})
    df
+
+Categorical Data
+----------------
+
+`Using categorical data type to store data more efficiently and consistently in multiple DataFrames
+<https://stackoverflow.com/questions/29709918/pandas-and-category-replacement/29712287#29712287>`
+
+`More information about categorical data <http://pandas.pydata.org/pandas-docs/stable/categorical.html>`
+
+.. ipython:: python
+
+   import pandas as pd
+   import string
+   import numpy as np
+   from pandas.api.types import CategoricalDtype
+
+Create a numpy array of all the ascii letters
+
+.. ipython:: python
+
+   np.random.seed(1234)
+   pd.set_option('max_rows',10)
+   uniques = np.array(list(string.ascii_letters))
+   uniques
```

Comment:
I would make this less verbose on the creation here

Reply:
How's this?
[UniqCats.pdf](https://github.com/pandas-dev/pandas/files/1636524/UniqCats.pdf)

Reply:
I'll try to simplify this